tests/postgres_tests/test_ranges.py

```diff
@@ -91,14 +91,76 @@ class TestQuerying(PostgreSQLTestCase):
 
     @classmethod
     def setUpTestData(cls):
+        cls.timestamps = [
+            datetime.datetime(year=2016, month=1, day=1),
+            datetime.datetime(year=2016, month=1, day=2, hour=1),
+            datetime.datetime(year=2016, month=1, day=2, hour=12),
+            datetime.datetime(year=2016, month=1, day=3),
+        ]
+
+        cls.dates = [
+            datetime.date(year=2016, month=1, day=1),
+            datetime.date(year=2016, month=1, day=2),
+            datetime.date(year=2016, month=1, day=3),
+            datetime.date(year=2016, month=1, day=4),
+        ]
+
         cls.objs = [
             RangesModel.objects.create(ints=NumericRange(0, 10)),
             RangesModel.objects.create(ints=NumericRange(5, 15)),
             RangesModel.objects.create(ints=NumericRange(None, 0)),
             RangesModel.objects.create(ints=NumericRange(empty=True)),
-            RangesModel.objects.create(ints=None),
+            RangesModel.objects.create(
+                ints=None, dates=(cls.dates[0], cls.dates[3]), timestamps=(cls.timestamps[0], cls.timestamps[3])
+            ),
         ]
 
+    def test_datetime_range_contains_date(self):
```

Comment:
Can the new tests go in a separate test case? I think that would make things simpler.